docs/joss_paper/paper.md

```diff
@@ -0,0 +1,120 @@
+
+## The software
+`DeepReg` is unit tested, and includes both technical documentation and various tutorials to introduce users to various use cases using open-sourced datasets. The continuous integration platform TravisCI (https://travis-ci.org/) is used together with a cutomised contributor guideline to facilitate contributions.
+
+
+# Example applications
+`DeepReg` provides a collection of demonstrations, _DeepReg Demos_, using open-accesible data with real clinical applications.
+
+## Paired images
+If images are available in pairs, two dataset loaders are currently available for paired images with- and without labels. Many clinical application for tracking organ motion and other temporal changes between two time points require _intra-subject_ _single-modality_ image registration. Registering lung CT images for the same patient, acquired at expritory and inspritory, is such an example of both unsupervised (without labels) and combined supervision (with segmentation of anatomical structures). Furthermore, registering prostate MR, aquired before surgery, and the intra-operative trasnrectal ultrasound images is an example of weakly-supervised training for multimodal image registration.
+
+## Unpaired images
+Unpaired images are found in applications such as single-modality inter-subject registration. Two dataset loaders for labelled and unlabelled image data. Registering different MR images of brain from different subjects has played a fundamental role in population studies in neuroimaging. An example registering 3D ultrasound images from different surgical patients are also provided, especially to demonstrate the support for experiments such as cross-validation in DeepReg.
+
+## Grouped images
+Unpaired images can also be grouped in applications such as single-modality intra-subject registration, in which each subject has multiple images acquired, e.g. at different time points. Two dataset loaders are implemented for grouped image data, with- and without labels. Paired dataset loader is a special case of the grouped dataset loader. Prostate logitudinal MR is proposed to track the progression of prostate cancer patients at an early stage during the active surveilence programme [@yang2020longitudinal]. One DeepReg demo illustrates how the grouped dataset loader can be used for deep-learning based feature reigstration by aligning intra-patient organ masks.
```

Comment:
so the grouped image demo has to be ready before the submission?

Reply:
yes